lib/utils.js

```diff
@@ -2,21 +2,9 @@
 
 var bind = require('./helpers/bind');
 
-/*global toString:true*/
-
 // utils is a library of generic helper functions non-specific to axios
 
-var toString = Object.prototype.toString;
-
-/**
- * Determine if a value is an Array
- *
- * @param {Object} val The value to test
- * @returns {boolean} True if value is an Array, otherwise false
- */
-function isArray(val) {
-  return toString.call(val) === '[object Array]';
-}
+var _toString = Object.prototype.toString;
```

Comment:
Solid PR. The only tiny thing I can mention is keeping the old name `toString`.

Reply:
I think I did this to remove the `/* global ` directive. I've reverted to the old name per your suggestion :+1: